content/en/docs/getting-started-guides/fedora/flannel_multi_node_cluster.md

```diff
@@ -38,7 +38,7 @@ Choose an IP range that is *NOT* part of the public IP address range.
 Add the configuration to the etcd server on fed-master.
 
 ```shell
-etcdctl set /coreos.com/network/config < flannel-config.json
+etcdctl put /coreos.com/network/config < flannel-config.json
```

Comment:
This is really about adopting the etcdctl v3.x API (and that's fine, etcd 3 came out about 3 years ago).

Reply:
@kbhawkey done!